web_client/src/store/index.ts

```diff
@@ -231,10 +231,11 @@ function queueLoadScan(scan, loadNext = 0) {
         ];
       }
     } else {
-      let newIndex = scansInSameExperiment.indexOf(scan.id);
+      let newIndex = scansInSameExperiment.indexOf(scan.id) + 1;
```

Comment:
I assume the `+ 1` here is because we want the index after the current scan?

Reply:
Yes, It should have gone in the last commit.